src/SeleniumLibrary/locators/elementfinder.py

```diff
@@ -76,6 +77,20 @@ def find(self, locator, tag=None, first_only=True, required=True, parent=None):
             )
         if self._is_webelement(locator):
             return locator
+
+        multi_locator_pattern = '.* >> (identifier|id|name|xpath|dom|link|partial link|css|class|jquery|sizzle|tag|scLocator)(:|=).*'
+
+        if not isinstance(locator, list) and Matcher(multi_locator_pattern, regexp=True).match(locator):
```

Comment:
I would assume that it splitting could go separate method, at least it would be easier to test.

Reply:
Sure, I will extract the code to check for multiple locators and splitting to a separate method.